src/extensions/stores/cluster-store.ts

```diff
@@ -16,16 +17,21 @@ export class ClusterStore extends Singleton {
 
   /**
    * Active cluster id
+   *
+   * @deprecated use `workspaceStore.activeCluster`
    */
   get activeClusterId(): string {
-    return internalClusterStore.activeCluster;
+    console.warn("get Store.ClusterStore.activeClusterId is deprecated. Use workspace.activeCluster");
+
+    return internalWorkspaceStore.currentWorkspace.activeClusterId;
   }
 
   /**
    * Set active cluster id
+   * @deprecated use navigate
    */
   set activeClusterId(id : ClusterId) {
-    internalClusterStore.activeCluster = id;
+    console.warn("Store.ClusterStore.activeClusterId is deprecated. Use LensExtension.navigate()");
```

Comment:
So the previous code was basically completely broken so I doubt that anyone was really using it and I don't think we need to keep the behaviour. This wrapper class is marked as `@beta` anyway. I kept this setter (marking it as `@deprecated`) so that things will still build but I don't really think we need to maintain the broken code. 

I can be persuaded to the contrary, that is just my current position.

Reply:
You're right, it's no big deal.  I leave it up to you. 

But if it's easy to keep supporting that broken behaviour as deprecated then why not? Otherwise someone will say "I know it was broken, but now it's brokener". :)

Reply:
Okay sounds good. I am also now working on #1759 so in that PR I'll change what the deprecation message is.